homepages/assets/less/inn.less

```diff
@@ -10,105 +10,57 @@
     max-width: 100%;
   }
   #hero {
-    background-image: none;
-    background-color: #6D72E6;
+    background-image: url(../img/INN-Days-Hero.svg);
+    background-size: cover;
+    background-position: center center;
```

Comment:
this can just be `background-position: center;`

If you only specify one keyword, the other value will be "center"